.gitignore

```diff
@@ -34,6 +34,7 @@ captures/
 
 # IntelliJ
 *.iml
+.idea/
```

Comment:
By addition of this, all the following are useless, delete them

Reply:
I did not delete them to follow the .gitignore standard styles